GoogleForCreators/web-stories-wp
# Dashboard: Fix WordPress Admin Menu Scroll. Remove Squish for Dashboard Header.

## Summary

Fix container styles to ensure WordPress menu can scroll. Remove scroll hijacking on Dashboard to allow for a natural page flow.

## Relevant Technical Choices



## To-do
Need to remove all the squish code from the layout provider and remove `Layout.Squishable` cus we shouldn't need it any more

## User-facing changes

The WordPress Menu now can scroll if the browser height is short
The Dashboard Header is in-place and the page has a full natural scroll.

## Testing Instructions

1. Launch the Dashboard
2. Resize your browser window vertically so it is shorter
3. Scroll the page and verify the WordPress Admin Menu on the left can be scrolled

1. Launch the Dashboard
2. Scroll the My Stories view down to see the scroll to top button in the bottom right
3. Click that button and verify scrolling back to top works

Verify that there are no regressions when viewing:
- Telemetry Banner
- Toasts
- Any page (My Stories, Templates & Editor Settings)

#5647 
#5874
